tests/fixtures/dbt/import/manifest_empty_columns.json

```diff
@@ -669,102 +669,6 @@
                 "namespace": null
             }
         },
-        "test.jaffle_shop.relationships_orders_customer_id__customer_id__ref_customers_.c6ec7f58f2": {
```

Comment:
The `infer_primary_key` function determines which columns are primary keys based on the information in the `constraints` and `data_tests` in manifest.json. This json is for when a column is empty, but there are still tests for columns that are empty. Therefore, I am deleting the information about the tests because the primary key is mistakenly detected even though the column is empty.

- https://github.com/dbt-labs/dbt-core/blob/459d156e85089961359815a2e9938265a4c67c81/core/dbt/contracts/graph/nodes.py#L561-L568